src/server/worker/workerrequests.js

```diff
@@ -16,9 +16,9 @@ var Core = requireJS('common/core/coreQ'),
     constraint = requireJS('common/core/users/constraintchecker'),
```

Comment:
both. The majority of the code change is refactoring or consequences of it, but it contains small fixes as well. If anything would fall into enhancement category, that should be closely related to the refactor.